app/Resources/views/themes/tastefulplanet/block_item/video/default.html.twig

```diff
@@ -1,7 +1,15 @@
 <div class="col-md-12">
-    <video width="100%" controls>
-        <source src="{{ ez_field_value(content,"file")}}" type="video/mp4">
-        <source src="{{ ez_field_value(content,"file")}}" type="video/ogg">
+    {% set field_file = ez_field_value(content,"file") %}
+    {% set width = field_file.width is defined ? field_file.width ~ 'px' : '100%' %}
```

Comment:
To make sure the video is not bigger than the container, I would recommend to add this css somewhere:
```
video {
    width: 100%;
    height: auto;
}
```